lib/object-to-query-string.ts

```diff
@@ -1,7 +1,15 @@
-type QueryStringValue = string | number | boolean;
+type QueryStringValue = string | number | boolean | (string | number | boolean)[];
 
 const join = (obj: Record<string, QueryStringValue>): string => Object.keys(obj)
-  .map(key => `${key}=${encodeURIComponent(obj[key])}`)
+  .reduce((acc: string[], key) => {
+    const value = obj[key];
+
+    if (Array.isArray(value)) {
+      return acc.concat(value.map((item) => `${key}=${encodeURIComponent(item)}`));
+    }
+
+    return acc.concat(`${key}=${encodeURIComponent(value)}`);
```

Comment:
I decided to use `Array.concat` instead of spread operator to prevent adding of polyfills.